cmd/tealdbg/localLedger.go

```diff
@@ -187,6 +187,9 @@ func getAppCreatorFromIndexer(indexerURL string, indexerToken string, app basics
 	queryString := fmt.Sprintf("%s/v2/applications/%d", indexerURL, app)
 	client := &http.Client{}
 	request, err := http.NewRequest("GET", queryString, nil)
+	if err != nil {
+		return basics.Address{}, fmt.Errorf("application request error: %s", err)
```

Comment:
`%s` -> `%v`, since err is an object.

Reply:
I copied these line verbatim from a few lines down, but I think the modern thing to do is [`%w` here](https://blog.golang.org/go1.13-errors)

Reply:
correct. Using `%w` for errors is better.